lib/src/util/tested_expressions.dart

```diff
@@ -65,6 +65,7 @@ bool _sameOperands(String eLeftOperand, String bcLeftOperand,
   return sameOperandsSameOrder || sameOperandsInverted;
 }
 
+// ignore: camel_case_types
```

Comment:
Not a good one.  This case is what got me started down this road:

    class MockLineInfo_Location extends Mock implements LineInfo_Location {}

which makes sense to ignore IMO.

I've fixed the other ones though.

👍